python/cudf/cudf/_libxx/sort.pyx

```diff
@@ -17,11 +17,13 @@ from cudf._libxx.cpp.table.table cimport table
 from cudf._libxx.cpp.table.table_view cimport table_view
 from cudf._libxx.cpp.search cimport lower_bound, upper_bound
 from cudf._libxx.cpp.sorting cimport(
-    rank, rank_method, underlying_type_t_rank_method, sorted_order,
-    is_sorted as cpp_is_sorted
+    rank, rank_method, sorted_order, is_sorted as cpp_is_sorted
 )
 from cudf._libxx.cpp.types cimport order, null_order, include_nulls
 
+from libc.stdint cimport int32_t
+ctypedef int32_t underlying_type_t_rank_method
```

Comment:
This `ctypedef` should go into `_libxx/sort.pxd` so that it can be `cimport` by other modules if needed.

Reply:
Yea, you should create it and then `cimport underlying_type_t_rank_method` from there